cmd/scale.go

```diff
@@ -155,6 +143,24 @@ func (sc *scaleCmd) load(cmd *cobra.Command) error {
 		return errors.Wrap(err, "error parsing the api model")
 	}
 
+	if sc.containerService.Properties.IsAzureStackCloud() {
+		writeCustomCloudProfile(sc.containerService)
```

Comment:
@jackfrancis , this function will write azure.Environment into temp file and set the environment variable to consume it.